mq/src/share/cclient/io/Packet.cpp

```diff
@@ -1260,7 +1260,7 @@ Packet::setMessageBody(PRUint8 * const msgBodyArg,
 {
   CHECK_OBJECT_VALIDITY();
 
-  ASSERT( msgBodyArg >= 0 );
+  ASSERT( msgBodyArg >= reinterpret_cast<PRUint8 *>(0) );
```

Comment:
Given the error message
```
ordered comparison of pointer with integer zero (‘PRUint8*’ {aka ‘unsigned char*’} and ‘int’)
```
I think it's time to believe that `unsigned char*` can't be negative at any moment.
Hence - I suggest to drop this line.

Reply:
Yes, I also think it is reasonable to assume that and remove the ASSERT.